LibreNMS/OS/Powerconnect.php

```diff
@@ -62,6 +62,9 @@ public function discoverProcessors()
         } elseif (starts_with($device['sysObjectID'], '.1.3.6.1.4.1.674.10895.3024')) {
             d_echo("Dell Powerconnect 8024F");
             return $this->discoverVxworksProcessors('.1.3.6.1.4.1.674.10895.5000.2.6132.1.1.1.1.4.9.0');
+        } elseif (starts_with($device['sysObjectID'], '.1.3.6.1.4.1.674.10895.3065')) {
```

Comment:
`starts_with` accepts an array as the second arg so seeing as you are just returning the same as the one above then remove the new elseif and change the one above to:

```php
} elseif (starts_with($device['sysObjectID'], ['.1.3.6.1.4.1.674.10895.3024', '.1.3.6.1.4.1.674.10895.3065'])) {
```

Give that a try and update the PR.

Reply:
@laf thanks! Updated.